README.md

```diff
@@ -264,6 +265,13 @@ for a system-wide installation.
 this will install a _python_ library **pyccel** and a _binary_ called **pyccel**.
 Any required Python packages will be installed automatically from PyPI.
 
+### On a read-only system
+
+If the folder where pyccel is saved is read only without sudo privileges an additional command should be run:
+```sh
+sudo pyccel-init
+```
```

Comment:
Is this still true? Won't `setup.py` create and install the pickle files for us?

I suspect that this command is only needed if pyccel is updated.

Reply:
Fixed

Reply:
I believe that `setup.py` should install the pickle files for us. But I suspect it is impossible to be sure until we have tested by pulling from pypi

Reply:
I almost forgot, please update the version

Reply:
Fixed [4a4d0ab](https://github.com/pyccel/pyccel/pull/1081/commits/4a4d0ab36d32a612647c0d52476ae0b68dea91b0)